certbot/certbot/_internal/main.py

```diff
@@ -921,6 +922,62 @@ def update_account(config: configuration.NamespaceConfig,
     return None
 
 
+def fetch_account(config: configuration.NamespaceConfig,
+                   unused_plugins: plugins_disco.PluginsRegistry) -> Optional[str]:
+    """Fetch account info on the server.
+
+    :param config: Configuration object
+    :type config: configuration.NamespaceConfig
+
+    :param unused_plugins: List of plugins (deprecated)
+    :type unused_plugins: plugins_disco.PluginsRegistry
+
+    :returns: `None`
+    :rtype: None
+
+    """
+    # Portion of _determine_account logic to see whether accounts already
+    # exist or not.
+    account_storage = account.AccountFileStorage(config)
+    accounts = account_storage.find_all()
+
+    if not accounts:
+        return "Could not find an existing account to fetch."
```

Comment:
Maybe `... existing account for server {config.server}`?

It might help users who are trying to use the verb but don't realize they're running it for the wrong server.

Reply:
Should whe then also update unregister() and update_account()? I just c/p'd this part with s/update/fetch..

Reply:
Yes, if you agree that it's an improvement, let's do that.